fpdf/fonts.py

```diff
@@ -1,14 +1,9 @@
-#!/usr/bin/env python
-
 # Fonts:
 
-fpdf_charwidths = {"courier": {}}
-
-for i in range(256):
-    fpdf_charwidths["courier"][chr(i)] = 600
-    fpdf_charwidths["courierB"] = fpdf_charwidths["courier"]
-    fpdf_charwidths["courierI"] = fpdf_charwidths["courier"]
-    fpdf_charwidths["courierBI"] = fpdf_charwidths["courier"]
+fpdf_charwidths = {
+    name: {chr(i): 600 for i in range(256)}
```

Comment:
Why making this change?
With the new version, the code is less readable...
Sometimes a simple `for` loop is easier to read 😊

Reply:
The difference is, that in the original case the four subdicts were the same subdict references four times, which has probably no impact when read-only. Anyway, the three subdicts were created at each iteration. At least I'll create them only once. See my new commit.